RingbufferStoreTest.testRingbufferStoreAllAndReadFromMemory

```
java.lang.AssertionError: expected:<200> but was:<0>
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.failNotEquals(Assert.java:834)
    at org.junit.Assert.assertEquals(Assert.java:645)
    at org.junit.Assert.assertEquals(Assert.java:631)
    at com.hazelcast.ringbuffer.impl.RingbufferStoreTest.testRingbufferStoreAllAndReadFromMemory(RingbufferStoreTest.java:138)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-OracleJDK1.7/com.hazelcast$hazelcast/648/testReport/junit/com.hazelcast.ringbuffer.impl/RingbufferStoreTest/testRingbufferStoreAllAndReadFromMemory/

```
java.lang.AssertionError: expected:<200> but was:<0>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.ringbuffer.impl.RingbufferStoreTest.testRingbufferStoreAllAndReadFromMemory(RingbufferStoreTest.java:138)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast/1079/testReport/junit/com.hazelcast.ringbuffer.impl/RingbufferStoreTest/testRingbufferStoreAllAndReadFromMemory/

```
java.lang.AssertionError: expected:<200> but was:<0>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.ringbuffer.impl.RingbufferStoreTest.testRingbufferStoreAllAndReadFromMemory(RingbufferStoreTest.java:138)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-ZuluJDK8/com.hazelcast$hazelcast/393/testReport/junit/com.hazelcast.ringbuffer.impl/RingbufferStoreTest/testRingbufferStoreAllAndReadFromMemory/

--- a/hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferStoreTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferStoreTest.java
@@ -123,6 +123,7 @@ public class RingbufferStoreTest extends HazelcastTestSupport {
         final TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
         final HazelcastInstance instance = factory.newHazelcastInstance(config);
         final HazelcastInstance instance2 = factory.newHazelcastInstance(config);
+        warmUpPartitions(instance, instance2);
 
         // add items to both ring buffers (master and backup) and shut down the master
         final Ringbuffer<Object> ringbuffer = instance.getRingbuffer("testRingbufferStore");
@@ -213,6 +214,7 @@ public class RingbufferStoreTest extends HazelcastTestSupport {
         final TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(3);
         final HazelcastInstance instance1 = factory.newHazelcastInstance(config);
         final HazelcastInstance instance2 = factory.newHazelcastInstance(config);
+        warmUpPartitions(instance1, instance2);
 
         final String name = generateKeyOwnedBy(instance1);
         final Ringbuffer<Object> masterRB = instance1.getRingbuffer(name);

